source/qb64pe.bas

```diff
@@ -12574,19 +12574,30 @@ IF GenerateLicenseFile AND NOT NoExeSaved THEN
     makeline$ = makeline$ + " GENERATE_LICENSE=y"
 END IF
 
+'Clear nm output from previous runs
+FOR x = 1 TO ResolveStaticFunctions
+    IF LEN(ResolveStaticFunction_File(x)) THEN
+        s$ = tmpdir$ + "nm_output_" + StrReplace$(StrReplace$(ResolveStaticFunction_File(x), pathsep$, "."), ":", ".") + ".txt"
+        IF _FILEEXISTS(s$) THEN KILL s$
+    END IF
+NEXT x
+
 IF os$ = "WIN" THEN
 
     makeline$ = makeline$ + " OS=win"
 
     'resolve static function definitions and add to global.txt
     FOR x = 1 TO ResolveStaticFunctions
+        nm_output_file$ = tmpdir$ + "nm_output_" + StrReplace$(StrReplace$(ResolveStaticFunction_File(x), pathsep$, "."), ":", ".") + ".txt"
         IF LEN(ResolveStaticFunction_File(x)) THEN
 
             n = 0
-            SHELL _HIDE "cmd /c internal\c\c_compiler\bin\nm.exe " + CHR$(34) + ResolveStaticFunction_File(x) + CHR$(34) + " --demangle -g >internal\temp\nm_output.txt"
+            IF NOT _FILEEXISTS(nm_output_file$) THEN
+                SHELL _HIDE "cmd /c internal\c\c_compiler\bin\nm.exe " + CHR$(34) + ResolveStaticFunction_File(x) + CHR$(34) + " --demangle -g >" + CHR$(34) + nm_output_file$ + CHR$(34)
```

Comment:
I made an `AddQuotes$()` function which just adds the `CHR$(34)`s around a string, it might simplify these lines a bit if you use it.